### DISCUSSION TITLE
Can I add comments for each repo?

### DISCUSSION THREAD

**Discussion by advpetc** [community/community#12223] [Other Feature Feedback, Questions, & Ideas]
I see we can add some descriptions for the entire list, but would it be possible to add some notes for each repo I star?

**Comment by pandruszkow** [comment]
(Not an answer) This would be perfect. Currently I track this as a Markdown file in a Git repository, but it's very clunky. Adding comments would be very useful.

**Reply by ProfDau-su** [reply]
(Not an answer)  Yes, this is a feature I wanted as soon as I tried to use lists.
